README.md

```diff
@@ -397,30 +397,39 @@ You can customize the name of this table using the `--migrations-table` flag or
-| CliсkHouse                                                   |              :white_check_mark:              |                                           |                                                      |                     :white_check_mark:                      |                             :white_check_mark:                              |                            :white_check_mark:                            |
-
-> :eight_pointed_black_star: In theory these tools could be used with other languages, but a Go development environment is required because binary builds are not provided.
-
-_If you notice any inaccuracies in this table, please [propose a change](https://github.com/amacneil/dbmate/edit/main/README.md)._
+| PostgreSQL                                                   |    ✅    |   ✅    |      ✅       |        ✅        |       ✅       |     ✅      |    ✅    |
+| MySQL                                                        |    ✅    |   ✅    |      ✅       |        ✅        |       ✅       |     ✅      |    ✅    |
+| SQLite                                                       |    ✅    |   ✅    |      ✅       |        ✅        |       ✅       |     ✅      |    ✅    |
+| CliсkHouse                                                   |    ✅    |         |               |        ✅        |       ✅       |     ✅      |          |
+| **Other Notes**                                              |          |         |               |                  |                |             |    ⁵     |
+
+> ¹ In theory these tools could be used with other languages, but a Go development environment is required because binary builds are not provided.
+>
+> ² Sqitch uses three separate migration in folders: up, down, and verify, which validates that the changes were applied correctly. Transactions are not applied automatically in the files, but the generated templates by default include transactions that can be removed as required.
+>
+> ³ Sqitch migration files are not timestamped, but they are recorded in a plan file and can have prerequisites specified. Sqitch records all migration actions (forward or backward) in multiple tables in the `sqitch` namespace, which can be customized.
+>
+> ⁴ Sqitch is framework independent, but requires a Perl installation or the use of a Docker image to run (which contains the Perl installation).
+>
+> ⁵ Sqitch has support for Firebird, Exasol, Oracle, Snowflake, and Vertica. Adding additional databases is possible but non-trivial. Sqitch also has direct support for integrating pgTAP (PostgreSQL unit tests) and support for Perl's App::Template so that common migrations can be templated (if you have tables that have the same basic definition). There are many other features to Sqitch.
```

Comment:
I appreciate the detail in these footnotes, but I want to keep this table simple because it can get out date quickly. The primary purpose of this table is to answer "why did you create another database migration tool", not provide a detailed comparison (which would be better in a blog post).

- (2) can be removed, it seems like Sqitch uses sql files to just leave the checkmark
- (3) can be removed, it's too much detail about the functionality of a third party 
- (4) I think it's fine to leave the checkmark and remove the footnote - interpreted languages always need a language interpreter, so it's a bit different to the (1) Go footnote (which it looks like is actually out of date anyway)
- (5) can be removed